### DISCUSSION TITLE
Redis Cache usecase

### DISCUSSION THREAD

**Discussion by luwol03** [inventree/InvenTree#3390] [Q&A] [answered]
Hello,

First of all, I hope you don't feel annoyed by all my comments.

I read about the redis cache and asked me **what it is used** for and **if it's required/optional/recommend**. The only thing I could find is [this](https://inventree.readthedocs.io/en/latest/start/docker_prod/#redis-cache), but there is no explaination.

luwol03

**Comment by matmair** [comment, marked as answer]
@luwol03 redis is used as a database cache if available. Requests are cached so that follow ups are faster. You do not needed it, it just makes the deployment faster - especially in bigger deployments with a lot of users.
